apps/extension/src/ui/domains/Portfolio/AssetDetails/animations/monad/MonadAnimation.tsx

```diff
@@ -0,0 +1,57 @@
+import { classNames } from "@talismn/util"
+import { useEffect, useState } from "react"
+import { createPortal } from "react-dom"
+
+import { useScrollContainer } from "@talisman/components/ScrollContainer"
+
+import imgAnimation1 from "./monad-animated-1.gif"
+import imgAnimation2 from "./monad-animated-2.gif"
+
+export const MonadAnimation = () => {
+  const refContainer = useScrollContainer()
+  const [step, setStep] = useState(0)
+
+  useEffect(() => {
+    const timeouts: ReturnType<typeof setTimeout>[] = []
+
+    const addTimeout = (callback: () => void, delay: number) => {
+      const timeout = setTimeout(callback, delay)
+      timeouts.push(timeout)
+    }
+
+    setStep(1)
+
+    addTimeout(() => {
+      setStep(2)
+      addTimeout(() => setStep(3), 3000)
+    }, 2000)
```

Comment:
Nitpick but wdyt about this?

```suggestion
    addTimeout(() => setStep(2), 2000)
    addTimeout(() => setStep(3), 5000)
```